Resources/ServerInfo/Guidebook/Science/Robotics.xml

```diff
@@ -0,0 +1,35 @@
+<Document>
+# Robotics
+The science team has the ability to create several unique types of robots to suit the station's needs.
+
+## Exosuit Forge
+<Box>
+<GuideEntityEmbed Entity="ExosuitFabricator" Caption="Exosuit Fabricator"/>
+</Box>
+The metallic womb from which all robotic children rise, bit by bit. The [color=#a4885c]Exosuit Forge[/color] is where you'll be printing out most of the parts for your creations, such as:
+- Borg limbs
+- Mech parts and equipment
+- Proximity sensors
+
+## Bots
+<Box>
+<GuideEntityEmbed Entity="MobCleanBot" Caption="Cleanbot"/>
+</Box>
+[color=#a4885c]Bots[/color] are small, simple automata which can perform simple tasks, such as cleaning. They don't use power cells, but are quite fragile.
+
+## Mechs
+<Box>
+<GuideEntityEmbed Entity="MechRipley" Caption="Ripley APLU"/>
+</Box>
+[color=#a4885c]Mechs[/color] are giant behemoths of metal designed to carry out a variety of functions respective to their model. They are piloted by players and requre power cells to function.
+
+All mechs require assembly before usage; after slotting each part into the chassis, [color=#a4885c]Examine[/color] it to see how to proceed with construction.
+
+## Equipment
+<Box>
+<GuideEntityEmbed Entity="MechEquipmentGrabber" Caption="Hydraulic Clamp"/>
+</Box>
+A mech is useless without any equipment. [color=#a4885c]Equipment[/color] are tools which can only be used by a mech.
```

Comment:
this should be above the picture of the grabber since it describes mech equipment in general and not just the grabber

Reply:
fixed